.python-version

```diff
@@ -0,0 +1 @@
+3.4.1
```

Comment:
@dethos  it is just to save the python version somewhere in order to prevent conflicts in the future or to ease the setup of the project to a new dev, using for example https://github.com/yyuu/pyenv